device/virtual/x86_64-kvm_x86_64-r0/virtual_chassis/topology_nbr.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Find another place to move the virtual_chassis directory.

Reply:
Guohan pointed out the symlink is present under platform/vs/tests/conftest.py -> sonic-swss/tests/conftest.py.
Is it okay to move the virtual_chassis under platform/vs/tests/ ?

Reply:
we will address this in PR2.